templatefield/test/models.py

```diff
@@ -4,7 +4,11 @@
 
 
 class TemplatedText(models.Model):
-    value = fields.TemplateTextField()
+    value = fields.TemplateTextField(null=True)
```

Comment:
Yeah, I think it makes sense to test that things don't blow up with null values even though it isn't good practice to have nullable TextFields.